staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -58,6 +58,39 @@ func (t *azureDiskCSITranslator) TranslateInTreeStorageClassToCSI(sc *storage.St
 	return sc, nil
 }
 
+// TranslateInTreeInlineVolumeToCSI takes a Volume with AzureDisk set from in-tree
+// and converts the AzureDisk source to a CSIPersistentVolumeSource
+func (t *azureDiskCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Volume) (*v1.PersistentVolume, error) {
+	if volume == nil || volume.AzureDisk == nil {
+		return nil, fmt.Errorf("volume is nil or Azure Disk not defined on volume")
+	}
+
+	azureSource := volume.AzureDisk
+	pv := &v1.PersistentVolume{
+		Spec: v1.PersistentVolumeSpec{
+			PersistentVolumeSource: v1.PersistentVolumeSource{
+				CSI: &v1.CSIPersistentVolumeSource{
+					Driver:           AzureDiskDriverName,
+					VolumeHandle:     azureSource.DataDiskURI,
+					ReadOnly:         *azureSource.ReadOnly,
+					FSType:           *azureSource.FSType,
+					VolumeAttributes: map[string]string{},
+				},
+			},
+			AccessModes: []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce},
+		},
+	}
+
+	if *azureSource.CachingMode != "" {
+		pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskCachingMode] = string(*azureSource.CachingMode)
+	}
+	if *azureSource.FSType != "" {
```

Comment:
should this be `if azureSource.FSType != nil && *azureSource.FSType {`?

Reply:
Thinking of changing this block to:
```
        if azureSource.FSType != nil {
                pv.Spec.PersistentVolumeSource.CSI.FSType = *azureSource.FSType
                pv.Spec.PersistentVolumeSource.CSI.VolumeAttributes[azureDiskFSType] = *azureSource.FSType
        }
```
Applies to both inline portion in this PR as well as https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go#L83 . 

Not quite sure @andyzhangx why the FSType needs to be passed as a VolumeAttribute parameter on top of the field?